modules/telco-troubleshooting-general-connect-to-pod.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * edge_computing/day_2_core_cnf_clusters/troubleshooting/telco-troubleshooting-general-troubleshooting.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="telco-troubleshooting-general-connect-to-pod_{context}"]
+= Connecting to a pod
+
+You can directly connect to a currently running pod with the `oc rsh` command, which provides you with a shell on that pod.
+
+[IMPORTANT]
+====
+Only use the `oc rsh` command if you cannot connect to the node with the `oc debug` command.
```

Comment:
```suggestion
Use the `oc rsh` command only if you cannot connect to the node by using the `oc debug` command.
```
- Place "only" as close to the word or phrase it modifies as possible: https://www.ibm.com/docs/en/ibm-style?topic=grammar-adverbs-only
- The term "with" can sometimes introduce ambiguity.